### PR TITLE
Added scripts and text file

### PR DESCRIPTION
**Pull request by CyanID401**

### CODE REVIEW

**Review by vh78 — COMMENTED** [review]
Body is missing.

**Comment by CyanID401** [comment]
All changes are applied.

File: scripts.js [hunk 1]
```diff
@@ -0,0 +1 @@
+console.log('not empty')
```
Comment (vh78) [inline scripts.js]: You can add function definition.

**Review by vh78 — CHANGES_REQUESTED** [review]

**Comment by CyanID401** [comment]
All changes are applied.

**Review by vh78 — APPROVED** [review]
LGTM!

**Pull request merged**